ibm_cloud_sdk_core/authenticators/no_auth_authenticator.py

```diff
@@ -0,0 +1,26 @@
+# coding: utf-8
+
+# Copyright 2019 IBM All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .authenticator import Authenticator
+
+class NoAuthAuthenticator(Authenticator):
+    authentication_type = 'noauth'
```

Comment:
If we're changing everything to `NoAuth`, we should be consistent. This value should be `noAuth` as well.